src/server/assets/assetsFilter/index.test.js

```diff
@@ -0,0 +1,123 @@
+import assetsFilter from './index';
+
+describe('assetsFilter', () => {
```

Comment:
Will there ever be a case where we have two bundles for a service? Is it worth adding a test case for the future in terms of this. 

EG:
```
/static/js/main-d0ae3f07.e24ffe78.js
/static/js/yoruba-31ecd969.d4952cef.js
/static/js/vendor-b23bnb22.nmn32mn2.js
/static/js/yoruba-f9ca8911.836b5376.js
/static/js/vendor-1f20a385.ff6d3f55.js
```

Reply:
Line 70 is an example of this